### PR TITLE
Temitope card update

### PR DESCRIPTION
**Pull request by TemitopeAgbaje**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -146,6 +146,43 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________Dex's card END ________  -->
```
Comment (dilshankarunarathne) [inline index.html:148]: Please leave just one empty line before and one empty line after your code.

**Review by dilshankarunarathne — CHANGES_REQUESTED** [review]
Hello @TemitopeAgbaje, thank you for contributing to this project. 
But there are a couple of things that need to be fixed before we merge your pull request.
Please go through the instructions again and move your card to line 118, so it would be the most recently added card.
Once you are done, please get back to us. We will be able to review and merge your card ASAP.

File: index.html (hunk 1 above)
Comment (TemitopeAgbaje) [reply index.html]: HI @dilshankarunarathne  I have corrected this .. Please kindly review

**Comment by dilshankarunarathne** [comment]
@TemitopeAgbaje, Congratulations 🎉❤
Your pull request was merged. 
You will be able to see your card on the [project website](https://syknapse.github.io/Contribute-To-This-Project/).

**Pull request merged**